pyproject.toml

```diff
@@ -37,6 +37,12 @@ repository = "https://github.com/lazyscribe/lazyscribe"
 requires = ["setuptools>=64.0.0"]
 build-backend = "setuptools.build_meta"
 
+[project.entry-points."artifact_type"]
+base = "lazyscribe.artifacts.base.Artifact"
+joblib = "lazyscribe.artifacts.joblib.JoblibArtifact"
+json = "lazyscribe.artifacts.json.JSONArtifact"
+testartifact = "tests.conftest.TestArtifact"
```

Comment:
Can we exclude the test-specific handler from being distributed with the package?